create-qa-release/action.yml

```diff
@@ -0,0 +1,108 @@
+name: "PCO Release RC release"
+inputs:
+  GITHUB_TOKEN:
+    description: token for access
+    required: true
+  package-json-path:
+    description: "path to package.json"
+    required: false
+    default: "package.json"
+  yarn-version-command:
+    description: "command to run to update version"
+    required: false
+    default: "yarn version"
+  node-version:
+    description: "node version to use"
+    required: false
+    default: "20"
+runs:
+  using: "composite"
+  steps:
+    - name: Set Release Name
+      shell: bash
+      run: echo "RELEASE_NAME=qa-${{ github.event.issue.number }}" >> $GITHUB_ENV
+    - name: Get PR head ref
+      id: get-pr-ref
+      uses: actions/github-script@v5
+      with:
+        script: |
+          const issueUrl = context.payload.issue.pull_request.url;
+          const prNumber = issueUrl.split('/').pop();
+          const pr = await github.rest.pulls.get({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            pull_number: prNumber,
+          });
+          return pr.data.head.ref;
```

Comment:
I think you can get the PR base using just `${{ github.base_ref }}`.

Reply:
👀

Reply:
unfortunately that is not available on comment actions which is how we plan to trigger the action